src/go/privileged-service/pkg/manage/install.go

```diff
@@ -46,9 +45,12 @@ func InstallService(name, displayName, desc string) error {
 		return err
 	}
 	defer m.Disconnect()
+
+	UninstallService(name)
```

Comment:
Don't always uninstall the service; just try to recreate.

Reply:
Recreating the service is risky since it will not uninstall the event logger. If the event logger is not uninstalled correctly it will yield a registry error, that requires manually deleting entries from the registry. It's an ugly state to get into.